app/jobs/reports/quarterly_account_stats.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Reports
+  class QuarterlyAccountStats < BaseReport
+    REPORT_NAME = 'quarterly-account-stats'
+
+    def perform(report_date)
+      report = report_body(report_date - 90.days, report_date)
+      save_report(REPORT_NAME, report, extension: 'csv')
+    end
+
+    def report_body(start_date, end_date)
+      report = {}
+
+      transaction_with_timeout do
+        report[:start_date] = start_date.to_s
+        report[:end_date] = end_date.to_s
+
+        report[:deleted_users_all_time] = DeletedUser.count
+        report[:deleted_users_for_period] = deleted_user_count(start_date:, end_date:)
+
+        report[:users_all_time] = User.count
+        report[:users_for_period] = user_count(start_date:, end_date:)
+
+        report[:users_and_deleted_all_time] =
+          report[:deleted_users_all_time] + report[:users_all_time]
+        report[:users_and_deleted_for_period] =
+          report[:deleted_users_for_period] + report[:users_for_period]
+
+        report[:proofed_all_time] = Profile.where(active: true).count
+        report[:proofed_for_period] = idv_user_count(start_date:, end_date:)
+      end
+
+      CSV.generate do |csv|
+        csv << report.keys
+        csv << report.values
+      end
```

Comment:
I feel like we could be building the CSV row-by row as we go?

```ruby
CSV.generate do |csv|
  csv << ['metric', 'value'] # header row
  csv << ['start_date', start_date.to_s]
  csv << ['end_date', end_date.to_s]
  # ...
end
```
and then the spec, instead of doing a `csv[0].zip(csv[1])` we could do:

```ruby
metrics = csv.map { |row| [row['metric'], row['value'] }.to_h
```

Reply:
I don't feel strongly, but I pictured the output data with column headings across the top and values on the row below, rather than something more tabular.

@zachmargolis, is one preferred over the other?

<small>~Plus, how often do we get an excuse to use `.zip` on arrays?~</small>

Reply:
I don't feel super strongly, other than that as a consumer of spreadsheets, I don't like scrolling horizontally too much. I think this structure makes sense if we have values over time in the same spreadsheet.

Another thought is that if we parse the CSV with headers with your current structure, we cold do:

```ruby
csv = CSV.parse(str, headers: true)
row = csv.first
row.to_h
```


example:

```ruby
csv = CSV.parse(<<-STR, headers: true)
a,b,c
1,2,3
STR
row = csv.first
row.to_h
# => {"a"=>"1", "b"=>"2", "c"=>"3"}
```